src/completers/completeTypes/test.js

```diff
@@ -1,7 +1,7 @@
 import completeTypes from '.';
 
 describe('completeTypes', () => {
-  it('Completes from an array\'s element', () => {
+  it("Completes from an array's element", () => {
```

Comment:
why double quotes?

Reply:
So I don't have to use \' and it's easier to read, I think